lib/utils/get-prs-to-days.ts

```diff
@@ -0,0 +1,29 @@
+import differenceInDays from "date-fns/differenceInDays";
+
+interface GraphData {
+  x: number,
+  y: number;
+}
+
+const getPullRequestsToDays = (pull_requests: DbRepoPR[]): GraphData[] => {
+  const graphDays = pull_requests.reduce((days: { [name: string]: number }, curr: DbRepoPR) => {
+    const day = differenceInDays(new Date(), new Date(curr.updated_at));
+
+    if (days[day]) {
+      days[day]++;
+    } else {
+      days[day] = 1;
+    }
+
+    return days;
+  }, {});
+
+  const days: any[] = [];
```

Comment:
Is there a reason why this variable is defined under the `graphDays` function where it's been used?